src/modules/live-execution-tracker/live-execution-tracker.ts

```diff
@@ -0,0 +1,519 @@
+
+      return elementCanHaveAToken;
+    });
+
+    // Get all elements that already have a token and all that have an active token.
+    const elementsWithActiveToken: Array<IShape> = await this._getElementsWithActiveToken(allElements);
+    const elementsWithTokenHistory: Array<IShape> = await this._getElementsWithTokenHistory(allElements);
+
+    // Colorize the found elements and add overlay to those that can be started.
+    this._colorizeElements(elementsWithTokenHistory, defaultBpmnColors.green);
+    this._colorizeElements(elementsWithActiveToken, defaultBpmnColors.orange);
+    this._addOverlaysToUserAndManualTasks(elementsWithActiveToken);
+
+    // Get the elementIds of the elements with an active token and sort them alphabetically
+    this._previousElementIdsWithActiveToken = elementsWithActiveToken.map((element: IShape) => element.id).sort();
+
+    // Export the colored xml from the modeler
+    const colorizedXml: string = await this._exportXml(this._diagramModeler);
+    return colorizedXml;
+  }
+
+  private _addOverlaysToUserAndManualTasks(elements: Array<IShape>): void {
+    const liveExecutionTrackerIsNotAttached: boolean = !this._attached;
+    if (liveExecutionTrackerIsNotAttached) {
+      return;
+    }
+
+    const activeManualAndUserTasks: Array<IShape> = elements.filter((element: IShape) => {
+      const elementIsAUserOrManualTask: boolean = element.type === 'bpmn:UserTask'
+                                               || element.type === 'bpmn:ManualTask';
+
+      return elementIsAUserOrManualTask;
+    });
+
+    const activeManualAndUserTaskIds: Array<string> =  activeManualAndUserTasks.map((element: IShape) => element.id).sort();
+
+    const elementsWithActiveTokenDidNotChange: boolean = activeManualAndUserTaskIds.toString() === this._previousElementIdsWithActiveToken.toString();
+    const allActiveElementsHaveAnOverlay: boolean = activeManualAndUserTaskIds.length === Object.keys(this._overlays._overlays).length;
+
+    if (elementsWithActiveTokenDidNotChange && allActiveElementsHaveAnOverlay) {
+      return;
+    }
+
+    for (const elementId of this._elementsWithEventListeners) {
+      document.getElementById(elementId).removeEventListener('click', this._handleElementClick);
+    }
+
+    this._elementsWithEventListeners = [];
+    this._overlays.clear();
+
+    for (const element of activeManualAndUserTasks) {
+      this._overlays.add(element, {
+        position: {
+          left: -1,
+          top: -1,
+        },
+        html: `<div class="play-task-button-container" id="${element.id}"><i class="fas fa-play play-task-button"></i></div>`,
+      });
+
+      document.getElementById(element.id).addEventListener('click', this._handleElementClick);
+
+      this._elementsWithEventListeners.push(element.id);
+    }
+  }
+
+  private _handleElementClick: (event: MouseEvent) => void =
+    (event: MouseEvent): void => {
+      const elementId: string = (event.target as any).id;
```

Comment:
you know the type of the target element..

```suggestion
      const elementId: string = (event.target as HTMLDivElement).id;
```